### DISCUSSION TITLE
ChromeOS -> Chrome 126.0.6478.132 is not seeing hub to pair, but Edge is

### DISCUSSION THREAD

**Discussion by n-davey** [pybricks/support#1702] [Q&A]
Hi,
I have a hub with pybricks firmware installed.
Current Chrome on ChromeOS does not see the hub to pair to, but Edge on the same machine works fine.
Earlier version of Chrome (125 or earlier, I don't recall) did work.

Any thoughts or help on how this can be resolved?

**Comment by BertLindeman** [comment]
Hi,

On my windows 11, I use Google Chrome Version 126.0.6478.127 (Official Build) (64-bit) without any problem.

**Comment by laurensvalk** [comment]
Does it persist when you close the Edge browser? 

Do you see any errors in the console? You can open the developer console by pressing <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>i</kbd> and selecting the `Console` tab:

![image](https://github.com/pybricks/support/assets/12326241/89766755-0eaf-409f-9439-139ad00d4371)

**Reply by n-davey** [reply]
If I pair in Edge, then close Edge it drops connection to hub, if that's what you mean.

In Chrome console, I see no significant errors:
![Screenshot 2024-07-06 22 17 47](https://github.com/pybricks/support/assets/122193285/0393011f-f161-418d-a591-41c103e7fb7d)

This seems to be a Bluetooth issue with Chrome on ChromOS (Chromebook), as Edge sees hub?

**Reply by n-davey** [reply]
Chrome Device Log:
![Screenshot 2024-07-06 22 29 55](https://github.com/pybricks/support/assets/122193285/57c7377c-6114-4e1f-8d15-5a3f947d6fe1)

**Reply by dlech** [reply]
FWIW, it is working on my Chrombook with the same Chrome version. So it seems likely that it is just an issue with your computer.

Are you running Edge on the same chromebook? How was Edge installed, as an android app?

**Reply by n-davey** [reply]
Hmm thanks.. interesting.. Wonder what's changed then..

Yes Edge is on the same Chromebook, Android app installed via Play Store.

I found a test page for the Web Bluetooth API (https://googlechrome.github.io/samples/web-bluetooth/device-info-async-await.html?namePrefix=Pyb), ran this in Chrome and it can see the hub to pair with.

I deleted the data and reset the permissions for pybricks in Chrome, retried and still can't see hub to pair with, strange..

**Reply by dlech** [reply]
Maybe you tried to pair the device in the ChromeOS system settings instead of in the browser? From looking at the ChromeOS source code, it looks like devices paired this way will be filtered out in the browser. So you would have to unpair it via the ChromeOS settings first. 

Although, that doesn't seem likely if it is showing up on a generic page.

Source is here if anyone else wants to have a look: https://source.chromium.org/chromium/chromium/src/+/refs/tags/126.0.6478.132:device/bluetooth/chromeos/bluetooth_utils.cc;l=315;bpv=0;bpt=0

**Reply by n-davey** [reply]
Hi,
It's not paired in OS Bluetooth, double checked that to be sure.. :)

It's interesting that the hub is visible to be paired in one tab (chrome sample) but not from another tab (pybricks).. 

I'll keep tinkering.. there will be something simple somewhere that's breaking it..

**Comment by Turbopsych** [comment]
I have the exact same issue, did you find a solution?

**Reply by n-davey** [reply]
I never found a solution to the original issue. But, can report that doing a check today, the problem now seems to be resolved.
ChromeOS 128.0.6613.133 and Chrome is confirmed working again.
